src/index.html

```diff
@@ -22,6 +22,401 @@
+        </p>
+
+        <p class="card__item-code">Product code: 195434</p>
+
+        <div class="card__item-review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__item-price">
+          <p class="card__item-price-word">Price:</p>
+          <p class="card__item-price-amount">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+          data-qa="card-hover"
+        >
+          BUY
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          src="..\src\images\imac.jpeg"
+          alt="Screen photo"
+          class="card__picture"
+        />
+
+        <p class="card__item-model">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__item-code">Product code: 195434</p>
+
+        <div class="card__item-review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__item-price">
+          <p class="card__item-price-word">Price:</p>
+          <p class="card__item-price-amount">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+        >
+          BUY
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          src="..\src\images\imac.jpeg"
+          alt="Screen photo"
+          class="card__picture"
+        />
+
+        <p class="card__item-model">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__item-code">Product code: 195434</p>
+
+        <div class="card__item-review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__item-price">
+          <p class="card__item-price-word">Price:</p>
+          <p class="card__item-price-amount">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+        >
+          BUY
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          src="..\src\images\imac.jpeg"
```

Comment:
The image path uses backslashes (`\`). Consider using forward slashes (`/`) for web URLs to ensure compatibility across different environments.